### PR TITLE
Doc strings within 80 char limit for klabel.el

### PR DESCRIPTION
**Pull request by matsl**
## What

Doc strings within 80 char limit for klabel.el.

## Why

Extracted from longer PR for easier review.

### CODE REVIEW

File: kotl/klabel.el [hunk 1]
```diff
@@ -478,7 +483,7 @@ and the start of its contents."
 		  (kcell-view:next nil lbl-sep-len)))))
 
 (defun klabel-type:update-labels (current-cell-label)
-  "Update the labels of current cell, its following siblings and their subtrees if need be.
+  "Update the labels of current cell, its siblings and their subtrees if need be.
```
Comment (rswgnu) [inline kotl/klabel.el:486]:
```suggestion
  "Update the labels of current cell, its following siblings and their subtrees.
```

File: kotl/klabel.el [hunk 2]
```diff
@@ -615,8 +620,8 @@ N may be an integer or a string containing an integer."
 ;;; ************************************************************************
 
 (defun klabel:set (new-label &optional lbl-sep-len)
-  "Replace label displayed in cell at point with NEW-LABEL, which may be a different label type.
-Return NEW-LABEL string."
+  "Replace label displayed in cell at point with NEW-LABEL.
+The label type may be different.  Return NEW-LABEL string."
```
Comment (rswgnu) [inline kotl/klabel.el:624]:
```suggestion
The new label type may differ from the old type.
Return NEW-LABEL string."
```

**Review by rswgnu — CHANGES_REQUESTED** [review]

**Review by rswgnu — APPROVED** [review]

File: ChangeLog [hunk 3]
```diff
@@ -85,7 +86,7 @@
 2022-07-21  Mats Lidell  <matsl@gnu.org>
 
 * kotl/kexport.el:
-* kotl/kcell.el: Shorten docs strings to be within 80 char limit.
+* kotl/kcell.el: Shorten docs strings to be within 80 char limit
```
Comment (matsl) [inline ChangeLog:89]:
```suggestion
* kotl/kcell.el: Shorten docs strings to be within 80 char limit.
```

Comment (matsl) [reply ChangeLog:89]: keyboard slip? Revert this change.

**Pull request merged**